xapian-bindings/configure.ac

```diff
@@ -287,11 +291,31 @@ case $with_python$with_python3$with_php7$with_ruby$with_tcl$with_csharp$with_jav
   test -z "$with_ruby" && with_ruby=no
   test -z "$with_perl" && with_perl=no
   test -z "$with_lua" && with_lua=no
+  test -z "$with_go" && with_go=no
   ;;
 esac
 
 BINDINGS=
-
+if test no != "$with_go" ; then
+  AC_PATH_PROG([GO],["go"],[])
+  AC_MSG_CHECKING(go)
+  if test -n "$GO" ; then
+    AC_MSG_RESULT(yes)
```

Comment:
You don't need `AC_MSG_CHECKING` + `AC_MSG_RESULT` here - you just used `AC_PATH_PROG` which will have said `checking for go...` followed by the path to the found `go` or else `no` if none was found, so your extra messages are look odd and confusing.